### PR TITLE
Add org.eclipse.search split and undo.snapshot details to n&n

### PR DESCRIPTION
**Pull request by robstryker**

### CODE REVIEW

File: news/4.30/platform.html [hunk 1]
```diff
@@ -75,6 +75,22 @@ <h2>Themes and Styling </h2>
     <h2>General Updates </h2>
     </td>
   </tr>
+   <tr id="eclipse_search"> <!-- https://github.com/eclipse-platform/eclipse.platform.ui/pull/1054 -->
```
Comment (akurtakov) [inline news/4.30/platform.html:78]: platform.html is for visible changes. [platform_isv.html](https://github.com/eclipse-platform/www.eclipse.org-eclipse/blob/master/news/4.30/platform_isv.html) is the one for such news. Sorry for not telling you earlier about that.

**Comment by vogella** [comment]
Thanks @robstryker for this update, looks good except the location as @akurtakov already said.

**Comment by robstryker** [comment]
Updated

**Comment by mickaelistria** [comment]
Thanks, this looks good to me. Further improvement if needed can be processed in further PRs.

**Pull request merged**